woocommerce/woocommerce
# Docs: Creating new performance category and adding best practices

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Adding new folder for Performance, creating a new document for performance best practices for extensions, moving existing store-performance document into this category



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Read the new document and readme for the new folder to make sure they're well written and accurate.
2. Compare to the guidance in [Canonical Extensions Quality - MVP](https://docs.google.com/spreadsheets/d/11dSNGAyDHPIbSMd2hAgUqlCl5_uqjy824kFQhGwPmRM/edit#gid=1783599133) to make sure the performance recommendation is clear. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
